partcad/src/partcad/converters/stl_converter.py

```
@@ -0,0 +1,100 @@
+    """Check if the given shape is a solid."""
+    explorer = TopExp_Explorer(shape, TopAbs_SOLID)
+    return explorer.More()
+
+
+def simplify_shape(shape: TopoDS_Shape) -> TopoDS_Shape:
+    """Simplify the given shape using BRepBuilderAPI_Copy."""
+    copier = BRepBuilderAPI_Copy(shape)
+    return copier.Shape()
+
+
+def dump_shape(shape: TopoDS_Shape):
```

Ruff (0.8.2)

Missing return type annotation for public function `dump_shape`

Add return type annotation: `None`

(ANN201)